ioctl/newcmd/account/accountdelete.go

```diff
@@ -110,6 +110,23 @@ func NewAccountDelete(c ioctl.Client) *cobra.Command {
 						filePath = v.URL.Path
 						break
 					}
+
+					aliases := make(map[string]string)
```

Comment:
Is this change related to the issue?

Reply:
no I think it was part of the code while rebasing I will fix it